docs/reference/elements/py-config.md

```diff
@@ -24,7 +24,10 @@ The `<py-config>` element should be placed within the `<body>` element.
   lang = "python"
 </py-config>
 ```
-Note: `[[runtimes]]` is a TOML table. Make sure this is the last item within a py-config, as the properties created after it go into the runtimes object.
+
+```{note}
+`[[runtimes]]` is a TOML table. Make sure this is the last item within a py-config, as the properties created after it go into the runtimes object.
+```
```

Comment:
Technically, it's just that TOML tables have to go after top level keys. So you can do:
```
[[runtimes]]
...
[[fetch]]
...
you just can't have the top level keys after either.

Reply:
That's a great point, I just turned the note into a proper note haha
Do you think we should remove the note? 🤔